docs/CHANGELOG.rst

```diff
@@ -9,6 +9,15 @@ All notable changes to this project are documented in this file.
 Unreleased
 ==========
 
+Added
+-----
+* Add ``get_primary_bam`` utility function
+
+Changed
+-------
+* Replace `get_bam`` with ``get_primary_bam`` utility function in
```

Comment:
Describe how this affects users, not how you achieve it, e.g. "Use only primary bam files in ``macs`` helper function"